File: VB/WorkflowDemoEF.Module/Objects/Issue.vb
Imports DevExpress.Persistent.Base
Imports System.ComponentModel
Imports System
Imports System.ComponentModel.DataAnnotations
Imports DevExpress.ExpressApp
Namespace WorkflowDemoEF.Module.Objects
    <DefaultClassOptions, DefaultProperty("Subject"), ImageName("BO_Note")> _
    Public Class Issue
        Private privateId As Integer
        <Browsable(False)> _
        Public Property Id() As Integer
            Get
                Return privateId
            End Get
            Private Set(ByVal value As Integer)
                privateId = value
            End Set
        End Property
        Public Property Subject() As String
        Public Property Active() As Boolean
    End Class
End Namespace

File: CS/WorkflowDemoEF.Module/Objects/Issue.cs
using DevExpress.Persistent.Base;
using System.ComponentModel;
using System;
using System.ComponentModel.DataAnnotations;
using DevExpress.ExpressApp;
namespace WorkflowDemoEF.Module.Objects {
    [DefaultClassOptions]
    [DefaultProperty("Subject")]
    [ImageName("BO_Note")]
    public class Issue {
        [Browsable(false)]
        public int Id { get; private set; }
        public string Subject { get; set; }
        public bool Active { get; set; }
    }
}
